product_docs/docs/biganimal/release/getting_started/identity_provider/aws_sso.mdx

```diff
@@ -0,0 +1,69 @@
+---
+title: "Using AWS SSO as your identity provider"
```

Comment:
Use the full name in the title and introduce the acronym in the first instance in the text